### PR TITLE
ENH : package ensures CONDA_ETC_ env is defined

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: beamlines/CHX/configuration/build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ETC=$PREFIX/etc
+
+
+# set up
+echo "# mds configuration
+host: xf11idb-ioc1
+database: datastore
+port: 27017
+timezone: US/Eastern
+" > $ETC/mds.yml
```
Comment (danielballan) [inline beamlines/CHX/configuration/build.sh]: Why create another name? And an asymmetry between metadatastore and filestore?

File: beamlines/CHX/configuration/build.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/etc
+ETC=$PREFIX/etc
+
+
+# set up
+echo "# mds configuration
```
Comment (danielballan) [inline beamlines/CHX/configuration/build.sh]: Consider adding `# file initially generated by CHX/configuration conda recipe`

File: beamlines/CHX/configuration/build.sh (hunk 1 above)
Comment (tacaswell) [inline beamlines/CHX/configuration/build.sh]: you prefer 'metadatastore'?

Comment (danielballan) [inline beamlines/CHX/configuration/build.sh]: Yes

File: beamlines/CSX/configuration/build.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+mkdir -p $PREFIX/etc
+ETC=$PREFIX/etc
+
+
+# set up
+echo "# metadatastore configuration
+# installed by csx_configuration
+# DO NOT EDIT
+host: xf23id-broker
+database: datastore
```
Comment (danielballan) [inline beamlines/CSX/configuration/build.sh]: At CSX, this is called `datastore2`.

**Pull request merged**